### PR TITLE
Network command tests

### PR DESCRIPTION
**Pull request by rvykydal**
Adding support for setting up multiple NICs.
Adding support for getting ks from local (hypervisor) http server.
Adding a test and its variation using the new features.

### CODE REVIEW

File: scripts/run_one_ks.sh [hunk 1]
```diff
@@ -30,6 +30,8 @@
 # 3  - Test failed due to kernel panic
 # 77 - Something needed by the test doesn't exist, so skip
 
+set -x
+
```
Comment (rvykydal) [inline scripts/run_one_ks.sh]: Remove debugging option

File: onboot-activate.sh [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+# injected in initrd) network devices are not found in sysfs in the time of
+# parsing the kickstart.
+
+TESTTYPE="network"
+
+. ${KSTESTDIR}/functions.sh
+
+
+kernel_args() {
+    . ${tmpdir}/ks_url
```
Comment (rvykydal) [inline onboot-activate.sh]: This line does not belong here (only used for ks supplied by http)

File: onboot-activate-httpks.ks.in [hunk 3]
```diff
@@ -0,0 +1,73 @@
+#test name: vlan
```
Comment (rvykydal) [inline onboot-activate-httpks.ks.in]: Fix also this

File: onboot-activate-httpks.sh [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+    echo "network:default"
+}
+
+prepare() {
+    ks=$1
+    tmpdir=$2
+
+    # Copy the kickstart to a directory in tmpdir
+    mkdir ${tmpdir}/http
+    cp $(dirname ${ks})/onboot.ks ${tmpdir}/http/ks.cfg
```
Comment (rvykydal) [inline onboot-activate-httpks.sh]: cp $ks ${tmpdir}/http/ks.cfg

**Comment by rvykydal** [comment]
jenkins, test this please

**Comment by jkonecny12** [comment]
Thank you for the tests!

**Pull request merged**